plugin/pkg/managedseed/validator/admission.go

```diff
@@ -355,6 +357,20 @@ func (v *ManagedSeed) admitSeedSpec(spec *gardencore.SeedSpec, shoot *gardencore
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("settings", "verticalPodAutoscaler", "enabled"), spec.Settings.VerticalPodAutoscaler.Enabled, "seed VPA is not supported for managed seeds - use the shoot VPA"))
 	}
 
+	k8sVersion := semver.MustParse(shoot.Spec.Kubernetes.Version)
```

Comment:
Is this `MustParse` expected here? Do we also do this in other admission plugins?

Reply:
I now removed `MustParse` and added error handling.